plugins/CoreHome/templates/_dataTable.twig

```diff
@@ -59,6 +59,8 @@
                 <div class="pk-emptyDataTable">
                 {% if showReportDataWasPurgedMessage is defined and showReportDataWasPurgedMessage %}
                     {{ 'CoreHome_DataForThisReportHasBeenPurged'|translate(deleteReportsOlderThan) }}
+                {% elseif showPluginArchiveDisabled %}
+                    {{ 'CoreHome_DataForThisReportHasBeenDisabled'|translate('<a target="_blank" href="'~'https://matomo.org/faq/how-to-disable-archiving-the-segment-reports-for-specific-plugins'|e('html_attr') ~'">', '</a>') }}
```

Comment:
In case you output a translation in a twig template that contains html, you need to apply the `raw` filter, otherwise the HTML is outputted encoded.